Add tags on cards for people section

CleverFool77 opened the pull request:
Fixes #5707 part

![Screenshot from 2019-05-22 15-47-45](https://user-images.githubusercontent.com/26685258/58167427-8f7a8780-7ca9-11e9-9e51-cda7b40e3c4b.png)
![Screenshot from 2019-05-22 15-48-52](https://user-images.githubusercontent.com/26685258/58167428-90131e00-7ca9-11e9-978a-90c85ff0e467.png)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

CleverFool77 commented:
Hi @jywarren @gauravano I wanted to know that Is the method I followed for this is correct ? 
I need bit of help here.
Thanks !!

jywarren commented:
I think this is looking good -- do you think you could make the tags a bit more like they appear on https://publiclab.org/tags? This is awesome!

jywarren commented on app/views/users/list.html.erb:
```diff
@@ -38,9 +38,12 @@
-                    <div>History : <%= user.nodes.count %> <%= t('users.list.notes_and_edits') %></div>
-                    <div> Joined : <%= distance_of_time_in_words(user.created_at, Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %></div>
+                  <span id="tags">
+                  <% Tag.sort_according_to_followers(user.tags,"desc") %>
```
whoa this is cool! is this a Rails method?

jywarren commented on app/views/users/list.html.erb:
```diff
@@ -38,9 +38,12 @@
-                    <div> Joined : <%= distance_of_time_in_words(user.created_at, Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %></div>
+                  <span id="tags">
+                  <% Tag.sort_according_to_followers(user.tags,"desc") %>
+                  <% user.tags.limit(3).each do |tag| %>
```
Should we store `Tag.sort_according_to_followers` in a var like: 

```ruby
tags = Tag.sort_according_to_followers
```
then use
```ruby
<% tags[0..2].each do |tag| %>
```

that way your sorting will persist, and also we're not running a new query?

CleverFool77 replied:
yes Now, the sorting is persisting.
I've updated it.
Thanks !!!

CleverFool77 commented:
> I think this is looking good -- do you think you could make the tags a bit more like they appear on https://publiclab.org/tags? This is awesome!

Hi @jywarren The code I'm using for tags UI is same as used in all other ages. It's just due to my screenshot which seems to changes the color to bit purple from blue.

CleverFool77 commented:
I just updated the code and new UI is ; 
![Screenshot from 2019-05-24 11-42-36](https://user-images.githubusercontent.com/26685258/58306476-43e6eb80-7e19-11e9-84a9-e9a76a083cd7.png)

the color looks different but in actual its `badge-primary` only.
Thanks !!!

jywarren commented:
Oh ok! Good to know, thanks! Is this ready for final review and merge? Thank you!

jywarren commented:
Actually one more thing; i think @gautamig54 noticed this but Junction is set as the font for many parts of the page, but as per the Style Guide, we want to only use it for headings like `h1, h2, h3`. Would you mind adjusting this? Thank you so much!

CleverFool77 commented:
Hi @jywarren I just fixed the font bug. I removed the assignment of font is beginning for whole section. So now It's working only for headings.
Thanks !!!

CleverFool77 commented:
This is ready for merge. I guess.
Thanks !!!

CleverFool77 commented:
I'll update few more stuff.
Thanks !!

CleverFool77 commented:
I just noticed that the class which helps in stacking map above in smaller screen seems to be removed.
Maybe it was removed by someone whole resolving merge conflicts. 
I'll add it back then.

CleverFool77 commented:
Hi @jywarren I did the changes online as my local branch is mess but checked it locally too 
I guess Now this PR is ready for merge.
Thanks !!

jywarren commented:
Oh cool! Can you upload one more screenshot? (hopefully screenshots won't be as needed soon - see https://github.com/publiclab/plots2/pull/5320) Just to see how the fonts work? Thank you!

CleverFool77 commented:
> Oh cool! Can you upload one more screenshot? (hopefully screenshots won't be as needed soon - see #5320) Just to see how the fonts work? Thank you!

Hi @jywarren
Sorry I slept yesterday before this message.
Here is the scrrenshot.
![Screenshot from 2019-05-25 14-47-09](https://user-images.githubusercontent.com/26685258/58367453-06f42500-7efd-11e9-8d1c-b9ecfcf810f7.png)

CleverFool77 commented:
The new feature of having screenshot during system test is awesome.
Damn !!!!!!!!!!!!!!!!!!!!!

jywarren commented:
Awesome here! 😄
